src/libaktualizr-isotp/isotpsecondary.cc

```diff
@@ -137,6 +137,12 @@ bool IsoTpSecondary::putMetadata(const RawMetaPack& meta_pack) {
   return conn.Send(out);
 }
 
+data::ResultCode::Numeric install(const Target& target) {
+  (void)target;
+  // TODO: implement an image sending data by using ImageReader and IsoTpSecondary::sendFirmware
+  return data::ResultCode::Numeric::kOk;
+}
+
```

Comment:
Historically this was just reusing the default string sending, right? How hard is it to just replicate that here? I know we don't have a good way to test that at the moment.

Reply:
> Historically this was just reusing the default string sending, right? 
Yes, it's in IsoTpSecondary::sendFirmware. 

> How hard is it to just replicate that here?

It's easy. The problem is that nothing instantiates IsoTpSecondary nor tests it, the question is what the point in the functionality that is never triggered.

Reply:
just implemented it.